### PR TITLE
Run AES-GCM check concurrently.

### PR DESCRIPTION
**Pull request by bryce-shang**
This PR added new entry of AES-GCM formal verification. The new entry `SELECT_CHECK` executes AES-GCM verification with different `evp_cipher_update_len` concurrently.

Related SIM: CryptoAlg-630

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: SAW/proof/AES/verify-AES-GCM-quickcheck.saw [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+*/
+
+// The number of bytes of input given to the update function.
+let evp_cipher_update_len = 101;
```
Comment (apetcher-amazon) [inline SAW/proof/AES/verify-AES-GCM-quickcheck.saw:7]: 380 is a better quick check length. It exercises more of the code, and also stresses memory a bit more so we can find performance issues sooner.

Comment (bryce-shang) [reply SAW/proof/AES/verify-AES-GCM-quickcheck.saw:7]: I will revert this change to `380`.

Comment (bryce-shang) [reply SAW/proof/AES/verify-AES-GCM-quickcheck.saw]: Reverted.

**Review by apetcher-amazon — APPROVED** [review]

**Review by apetcher-amazon — APPROVED** [review]

**Comment by bryce-shang** [comment]
Merging this PR because AES-GCM check (with diff parameters) passed in CodeBuild aws-lc-aes-gcm:83203246-cb54-400a-9a5a-adaf1aefb0c5

**Pull request merged**